### PR TITLE
Explain CORS in Readme

### LINKED ISSUE #52: Trouble setting up development server
I'm having trouble setting up a development server according to https://github.com/zonemaster/zonemaster-gui/tree/develop#contribution. I get the error message "Zonemaster Backend is not available" (shown above the Domain name field on /domain_check).

I also tried updating the environment files in /src/environments/ to use a local backend API endpoint (current Backend develop branch), but the results look very similar to me.

#### Network traffic

Request URL: https://zonemaster.fr/api
Request method: OPTIONS
Remote address: 192.134.4.83:443
Status code: 200
Version: HTTP/1.1
Referrer Policy: no-referrer-when-downgrade
Request headers:
```
Host: zonemaster.fr
User-Agent: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:66.0) Gecko/20100101 Firefox/66.0
Accept: */*
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate, br
Access-Control-Request-Method: POST
Access-Control-Request-Headers: content-type
Referer: http://192.168.56.11:4200/domain_check
Origin: http://192.168.56.11:4200
Connection: keep-alive
Cache-Control: max-age=0
```

Response headers:
```
HTTP/1.1 200 OK
Date: Fri, 17 May 2019 09:49:55 GMT
Server: Apache/2.4.6 (CentOS) OpenSSL/1.0.2k-fips
Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
Content-Type: application/json
Keep-Alive: timeout=5, max=100
Connection: Keep-Alive
Transfer-Encoding: chunked
```

The response body is empty.

#### Console
```
[WDS] Disconnected! client:175
Please set the api endpoint dns-check.service.ts:17:6
Angular is running in the development mode. Call enableProdMode() to enable the production mode. core.js:3683
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://zonemaster.fr/api. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing).[Learn More]
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://zonemaster.fr/api. (Reason: CORS request did not succeed).[Learn More]
Object { headers: {…}, status: 0, statusText: "Unknown Error", url: null, ok: false, name: "HttpErrorResponse", message: "Http failure response for (unknown url): 0 Unknown Error", error: error }
dns-check.service.ts:46:10
Object { headers: {…}, status: 0, statusText: "Unknown Error", url: null, ok: false, name: "HttpErrorResponse", message: "Http failure response for (unknown url): 0 Unknown Error", error: error }
footer.component.ts:32:6
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://zonemaster.fr/api. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing).[Learn More]
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://zonemaster.fr/api. (Reason: CORS request did not succeed).[Learn More]
Object { headers: {…}, status: 0, statusText: "Unknown Error", url: null, ok: false, name: "HttpErrorResponse", message: "Http failure response for (unknown url): 0 Unknown Error", error: error }
dns-check.service.ts:46:10
ERROR Error: "Uncaught (in promise): HttpErrorResponse: {"headers":{"normalizedNames":{},"lazyUpdate":null,"headers":{}},"status":0,"statusText":"Unknown Error","url":null,"ok":false,"name":"HttpErrorResponse","message":"Http failure response for (unknown url): 0 Unknown Error","error":{"isTrusted":true}}"
    Angular 11
core.js:1448
    Angular 4
    RxJS 5
    Angular 14
```

### PR DESCRIPTION
**Pull request by pamasse**
Explain CORS in Readme
Fix #52

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,9 @@ Please always make the Pull Request against the develop branch.
     
 * Thank you for your contribution!
 
+> In development mode, you will certainly have to enable [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) on your browser.
```
Comment (matsduf) [inline README.md]: "you will certainly have to" --> "you probably have to"?

File: README.md [hunk 2]
```diff
@@ -72,6 +72,9 @@ Please always make the Pull Request against the develop branch.
...
+> The Cross-origin resource sharing of browsers block every AJAX request that not match the exact host, protocol, and port of your site.
```
Comment (matsduf) [inline README.md]: "The Cross-origin resource sharing of browsers block every AJAX request that not match the exact host, protocol, and port of your site." --> "The Cross-Origin Resource Sharing (CORS) performed by the browser blocks every AJAX request that does not match the exact host, protocol, and port of your site." ?

Comment (pamasse) [reply README.md]: better yes

**Comment by mattias-p** [comment]
@pamasse In #52 you listed some possible approaches to solving the problem. Do you think it would be appropriate to include such a list in the Contribution section? Or is that something every developer is expected to know? Speaking for myself I know what CORS is in general terms, but don't have any experience of dealing with it in a development context. But then I don't call myself a frontend developer either. I guess this a question of what we expect from the reader.

**Comment by pamasse** [comment]
@mattias-p In my point of view, it's something that is very common. When you develop a GUI for a service that are not on the same URL, you will see this issue. So, the message in the readme is sufficient since it reminds the developer to be careful. After, a quick check on google, or now here on github and you will see a solution.

**Pull request merged**